documentation/API_documentation/DeviceLocation_API.md

```diff
@@ -0,0 +1,99 @@
+# Overview
+The Location API provides programmable interface for developers and other users to specify 
+if given user equipment is within the given range(radial distance) from the given coordinates(longitude, latitude) on the Earth.
+## 1\. Introduction
+Logistics, gaming and many more sectors of industry define a need of getting user real location. Moreover, some of them 
+need this functionality to prevent frauds - to compare GPS location with real user equipment one.
+<img src="./resources/Location_overview.png" alt="LOC_VER" title="Location verification">
+
+## 2\. Quick Start
+Location API exposes only one endpoint, which can be used to determine if given user equipment is within the given radial distance from
+given location. To do so we need to pass following info in request body: 
+1. **ueId** - an object with four fields, each of them make possible to pass ueId in different format: externalId (some string), msisdn, ipv4Addr and ipv6Addr 
```

Comment:
externalId here is not "some string" but should reference https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/UE-Identification.md#external-identifier-format-of-the-gpsi